e2e-tests/development-runtime/cypress/integration/navigation/redirect.js

```diff
@@ -5,34 +5,38 @@ Cypress.on(`window:before:load`, win => {
 
 const runTests = () => {
   it(`should redirect page to index page when there is no such page`, () => {
-    cy.visit(`/redirect-without-page`).waitForRouteChange()
+    cy.visit(`/redirect-without-page`, {
+      failOnStatusCode: false,
```

Comment:
I left these in as in the future when we'd send `404` for the not found page those would fail